ecl/network/v2/load_balancers/results.go

```diff
@@ -0,0 +1,137 @@
+package load_balancers
+
+import (
+	"github.com/nttcom/eclcloud"
+	"github.com/nttcom/eclcloud/pagination"
+)
+
+type commonResult struct {
+	eclcloud.Result
+}
+
+// Extract is a function that accepts a result and extracts a Load Balancer resource.
+func (r commonResult) Extract() (*LoadBalancer, error) {
+	var s struct {
+		LoadBalancer *LoadBalancer `json:"load_balancer"`
+	}
+	err := r.ExtractInto(&s)
+	return s.LoadBalancer, err
+}
+
+// CreateResult represents the result of a create operation. Call its Extract
+// method to interpret it as a Load Balancer.
+type CreateResult struct {
+	commonResult
+}
+
+// GetResult represents the result of a get operation. Call its Extract
+// method to interpret it as a Load Balancer.
+type GetResult struct {
+	commonResult
+}
+
+// UpdateResult represents the result of an update operation. Call its Extract
+// method to interpret it as a Load Balancer.
+type UpdateResult struct {
+	commonResult
+}
+
+// DeleteResult represents the result of a delete operation. Call its
+// ExtractErr method to determine if the request succeeded or failed.
+type DeleteResult struct {
+	eclcloud.ErrResult
+}
+
+// Interface represents a attached interfaces by Load Balancer.
+type Interface struct {
+	ID                  string `json:"id"`
+	IPAddress           string `json:"ip_address"`
+	Name                string `json:"name"`
+	NetworkID           string `json:"network_id"`
+	SlotNumber          int    `json:"slot_number"`
+	Status              string `json:"status"`
+	Type                string `json:"type"`
+	VirtualIPAddress    string `json:"virtual_ip_address"`
+	VirtualIPProperties string `json:"virtual_ip_properties"`
+}
+
+// Interface represents a attached interfaces by Load Balancer.
+type SyslogServer struct {
+	ID          string `json:"id"`
+	IPAddress   string `json:"ip_address"`
+	LogFacility string `json:"log_facility"`
+	LogLevel    string `json:"log_level"`
+	Name        string `json:"name"`
+	PortNumber  int    `json:"port_number"`
+	Status      string `json:"status"`
+}
+
+// LoadBalancer represents a Load Balancer. See package documentation for a top-level
+// description of what this is.
+type LoadBalancer struct {
+
+	// AdminPassword is admin's password
+	AdminPassword string `json:"admin_password"`
+
+	// AdminUsername is admin's username
+	AdminUsername string `json:"admin_username"`
+
+	// AvailabilityZone is one of the Virtual Server (Nova)’s availability zone.
+	AvailabilityZone string `json:"availability_zone"`
+
+	// Description is description
+	DefaultGateway string `json:"default_gateway"`
```

Comment:
Default value of default_gateway is null. So you need to distinguish between nil and blank string
```suggestion
	DefaultGateway *string `json:"default_gateway"`
```